Neos.Flow/Classes/Mvc/ActionResponse.php

```diff
@@ -196,7 +196,7 @@ public function getStatusCode(): int
      */
     public function getContentType(): string
     {
-        return $this->contentType;
+        return $this->contentType ?? '';
```

Comment:
That's just semantic correction, since the return type hint was already in place and would have led to an fatal error if contentType was unset.

Reply:
In 7.0 the return type hint has been changed (https://github.com/neos/flow-development-collection/blob/32149adc310623c9e43e6849128d8248a6e536cc/Neos.Flow/Classes/Mvc/ActionResponse.php#L251) – is it a good idea to do "" vs NULL here?

Reply:
I would suggest to omit this unrelated(?) change from this PR unless we're 100% certain about it

Reply:
See https://github.com/neos/flow-development-collection/pull/2180/files#r550798576 - this is a semantic/typehint error and psalm complains about it. So we need to deal with this one way or another (adjust return type hint or use null coalescing). I still see no value in the nullability of this getter though. It would just be a technical necessity due to the missing property initialization. For all means and purposes '' is equivalent to `null`, but as of the addition of the `hasContentType()` check method that prevents calling `getContentType()` when it would be null.

Reply:
```suggestion
        return $this->contentType;
```
Removing this for now as of the `hasContentType()` check and moving this to an own PR where we can discuss if this getter should be nullable or not.